app/models/consultation.rb

```diff
@@ -363,6 +363,10 @@ def period_days
     end
   end
 
+  def needs_site_visit?
+    planning_application.prior_approval? ? neighbour_responses.objection.any? : true
+  end
```

Comment:
Is there a way we could use the `features` attribute on application type instead of checking directly. This will help when we scope out the features per application type when working on the global admin. Currently we have permitted development rights and planning conditions on the attribute